### PR TITLE
Update Lists Exercise Instructions to Improve Grammar

### PR DESCRIPTION
**Pull request by tombiju**
Numerous minor grammar fixes to the instructions

### CODE REVIEW

File: exercises/lists/en.md [hunk 1]
```diff
@@ -69,6 +69,6 @@ Try to write your own list. Please create new file and create an unordered list
 - Four
 - Five
 
-If you have done, please verify your solution.
+When you are done, please verify your solution with: how-to-markdown verify [yourFileName].md
```
Comment (denysdovhan) [inline exercises/lists/en.md]: Please, remove `with: how-to-markdown verify [yourFileName].md` — these lines are inserted by workshopper automatically.

**Comment by denysdovhan** [comment]
LGTM, but, please, fix a minor problem.

**Comment by tombiju** [comment]
ok thanks

**Comment by denysdovhan** [comment]
Merged and published as `v1.0.6`! 
Thank you!

**Pull request merged**